src/routes/files/index.js

```diff
@@ -8,6 +8,6 @@ const router = express.Router();
  */
 
 router.post('/', uploadHandler);
-router.delete('/:reportId/:fileId', deleteHandler);
+router.delete('/:reportId?/:fileId?', deleteHandler);
```

Comment:
Does this mean that both parameters are now optional? What's the use case for allowing those calls to be directed to the delete handler rather than 404ing?

Reply:
No, the parameters aren't optional. My reasoning was that if either or both are omitted it is more useful to get a 400 saying what was missing than a 404.